src/main/java/com/powsybl/openloadflow/network/impl/AbstractImpedantLfBranch.java

```diff
@@ -318,6 +336,152 @@ public void setClosedI2(Evaluable closedI2) {
         this.closedI2 = Objects.requireNonNull(closedI2);
     }
 
+    @Override
+    public void addAdditionalOpenP1(Evaluable openP1) {
+        additionalOpenP1.add(openP1);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalOpenP1() {
+        return additionalOpenP1;
+    }
+
+    @Override
+    public void addAdditionalClosedP1(Evaluable closedP1) {
+        additionalClosedP1.add(closedP1);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalClosedP1() {
+        return additionalClosedP1;
+    }
+
+    @Override
+    public void addAdditionalOpenQ1(Evaluable openQ1) {
+        additionalOpenQ1.add(openQ1);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalOpenQ1() {
+        return additionalOpenQ1;
+    }
+
+    @Override
+    public void addAdditionalClosedQ1(Evaluable closedQ1) {
+        additionalClosedQ1.add(closedQ1);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalClosedQ1() {
+        return additionalClosedQ1;
+    }
+
+    @Override
+    public void addAdditionalOpenP2(Evaluable openP2) {
+        additionalOpenP2.add(openP2);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalOpenP2() {
+        return additionalOpenP2;
+    }
+
+    @Override
+    public void addAdditionalClosedP2(Evaluable closedP2) {
+        additionalClosedP2.add(closedP2);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalClosedP2() {
+        return additionalClosedP2;
+    }
+
+    @Override
+    public void addAdditionalOpenQ2(Evaluable openQ2) {
+        additionalOpenQ2.add(openQ2);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalOpenQ2() {
+        return additionalOpenQ2;
+    }
+
+    @Override
+    public void addAdditionalClosedQ2(Evaluable closedQ2) {
+        additionalClosedQ2.add(closedQ2);
+    }
+
+    @Override
+    public List<Evaluable> getAdditionalClosedQ2() {
+        return additionalClosedQ2;
+    }
+
+    @Override
+    public void removeEvaluable(Evaluable evaluable) {
+        if (p1 == evaluable) {
+            p1 = NAN;
```

Comment:
It is weird that these lines are not covered...

Reply:
Covered!